tests/unit/expected_upf.json

```diff
@@ -14,7 +14,7 @@
   },
   "enable_notify_bess": true,
   "gtppsc": true,
-  "hwcksum": false,
+  "hwcksum": true,
```

Comment:
When adding this, I added a test that when the config was changed to `True`, it updated the rendered config. It should probably be reversed now also.

Reply:
Good catch. 
Done.